lib/models/github.dart

```diff
@@ -326,13 +326,47 @@ class GithubUserOrganizationItem {
       _$GithubUserOrganizationItemFromJson(json);
 }
 
+@JsonSerializable(fieldRename: FieldRename.snake)
+class GistFiles {
+  GistFiles({
+    this.filename,
+    this.size,
+    this.rawUrl,
+    this.type,
+    this.language,
+    this.truncated,
+    this.content,
+  });
+  String filename;
+  int size;
+  String rawUrl;
+  String type;
+  String language;
+  bool truncated;
+  String content;
+
+  factory GistFiles.fromJson(Map<String, dynamic> json) =>
+      _$GistFilesFromJson(json);
+}
+
 @JsonSerializable(fieldRename: FieldRename.snake)
 class GithubGistsItem {
-  int id;
-  bool isFork;
-  bool isPublic;
-  String name;
+  String id;
+  String description;
+  bool public;
+  Map<String, GistFiles> files;
+  GithubEventUser owner;
+  List<dynamic> get fileNames {
+    var filenames = [];
```

Comment:
```suggestion
    List<GistFiles> filenames = [];
```

Reply:
@shreyas1599 I checked it at my local machine and the error seems from here.

Reply:
@pd4d10 is the object tree building for you? This seems to fix the type error. I'm trying to fix the `object type error` now.

Reply:
Hi, I've pushed some changes. It works on my local machine. You could check if it works for you.

Reply:
Hi, these were the same changes I made. I pulled just in case I missed something. Still doesn't work.